spec/lib/facilities/ppms/response_spec.rb

```diff
@@ -0,0 +1,111 @@
+
+describe Facilities::PPMS::Response do
+  let(:provider_locator_body) do
+    [
+      {
+        'Miles' => 7.47076307768083,
+        'Minutes' => 11.5083333333333,
+        'ProviderName' => 'DOCTOR PARTNERS OF NORTHERN VIRGINIA',
+        'ProviderSpecialty' => 'Allergy & Immunology   ',
+        'SpecialtyCode' => '207K00000X',
+        'CareSite' => '4229 LAFAYETTE CENTER DR,UNIT 1760,CHANTILLY,VA,20151',
+        'CareSiteAddress' => '123RANDOM ST,CHANTILLY,VA 20151',
+        'CareSiteAddressCity' => 'MANASSAS',
+        'CareSiteAddressStreeet' => '123 RANDOM STREET',
+        'CareSiteAddressState' => 'VA',
+        'CareSiteAddressZipCode' => '22030',
+        'WorkHours' => nil,
+        'ProviderGender' => 'NotSpecified',
+        'ProviderNetwork' => 'Provider Agreement',
+        'NetworkId' => 8,
+        'ProviderAcceptingNewPatients' => false,
+        'ProviderPrimaryCare' => false,
+        'QualityRanking' => 0,
+        'ProviderIdentifier' => '1700950045',
+        'Latitude' => 45.5,
+        'Longitude' => -122.5
+      },
+      {
+        'Miles' => 59.2995756799325,
+        'Minutes' => 60.97,
+        'ProviderName' => 'DOCTOR PARTNERS OF SHENANDOAH VALLEY',
+        'ProviderSpecialty' => 'Allergy & Immunology   ',
+        'SpecialtyCode' => '207K00000X',
+        'CareSite' => '1828 W PLAZA DR,WINCHESTER,VA,22601',
+        'CareSiteAddress' => '1828 W PLAZA DR WINCHESTER,VA 22601',
+        'CareSiteAddressCity' => 'MANASSAS',
+        'CareSiteAddressStreet' => '123 RANDOM STREET',
+        'CareSiteAddressState' => 'VA',
+        'CareSiteAddressZipCode' => '22030',
+        'CareSitePhoneNumber' => '(888) 444-1234',
+        'WorkHours' => nil,
+        'ProviderGender' => 'NotSpecified',
+        'ProviderNetwork' => 'Provider Agreement',
+        'NetworkId' => 8,
+        'ProviderAcceptingNewPatients' => false,
+        'ProviderPrimaryCare' => false,
+        'QualityRanking' => 0,
+        'ProviderIdentifier' => '1427435759',
+        'Latitude' => 38.86787,
+        'Longitude' => -78.17305
+      },
+      {
+        'Miles' => 59.5528017001683,
+        'Minutes' => 61.8516666666667,
+        'ProviderName' => 'HEALTH CONSULTANTS OF VIRGINIA',
+        'ProviderSpecialty' => 'Ophthalmology',
+        'SpecialtyCode' => '207W00000X',
+        'CareSite' => '420 W JUBAL EARLY DRIVE STE 203,WINCHESTER,VA,22601',
+        'CareSiteAddress' => '420 W JUBAL EARLY DRIVE STE 203 WINCHESTER,VA 22601',
+        'WorkHours' => nil,
+        'ProviderGender' => 'NotSpecified',
+        'ProviderNetwork' => 'Provider Agreement',
+        'NetworkId' => 8,
+        'ProviderAcceptingNewPatients' => false,
+        'ProviderPrimaryCare' => false,
+        'QualityRanking' => 0,
+        'ProviderIdentifier' => '1114172319',
+        'Latitude' => 39.16996,
+        'Longitude' => -78.18052
+      }
+    ]
+  end
+
+  let(:response_body) { File.read('spec/fixtures/facilities/response_body.rb') }
```

Comment:
It looks like this fixture data comes directly from a VCR cassette.
These are unit tests of a class used to parse a response (i.e. a request to the external API isn't made here), but what do we think about just using the data from the VCR cassette directly?
That way if the VCR cassette is updated, and the results change it'll let us know we probably need to revisit this response object class too.
```suggestion
  let(:response_body) { JSON.parse(YAML.load_file('spec/support/vcr_cassettes/facilities/va/ppms.yml')['http_interactions'][3]['response']['body']['string'])['value'][0] }
```
Same would apply for the `provider_locator_body` fixture.

Interested to hear opinions. @johnpaulashenfelter

Reply:
@jholton, yes I like that idea better! Reduces repeated code and will get updated when the vcr cassette is updated as well.